### ISSUE TITLE
Tactic call becomes sexp with "[XXX ser_gen]"

### ISSUE THREAD

**Issue by palmskog** [ejgallego/coq-serapi#66]
When running `sercomp`, arguments to the built-in tactic `specialize` become sexps which contain `"[XXX ser_gen]"`. Such sexps can't be deserialized properly, which is a problem in the context I'm using serapi.

[Here](https://gist.github.com/palmskog/d8ff432a295d8833c62bab71e0fdfd41) is an example extracted from a real project we used.

Specifically, the line with the tactic call:
```
specialize (H a a).
```
gets converted into a sexp:
```
 (VernacExpr()(VernacExtend(VernacSolve 0)((GenArg raw(OptArg(ExtraArg ltac_selector))())(GenArg raw(OptArg(ExtraArg ltac_info))())(GenArg raw(ExtraArg tactic)(TacAlias((((fname(InFile Dedup_fail.v))(line_nb 44)(bol_pos 1015)(line_nb_last 44)(bol_pos_last 1015)(bp 1021)(ep 1039)))((Kername(MPfile(DirPath((Id Notations)(Id Init)(Id Coq))))(DirPath())(Id specialize_#_4C69D564))((TacGeneric(GenArg raw(ExtraArg constr_with_bindings)("[XXX ser_gen]"raw(ExtraArg constr_with_bindings)))))))))(GenArg raw(ExtraArg ltac_use_default)false))))
```
The key occurrence is
```
("[XXX ser_gen]"raw(ExtraArg constr_with_bindings))
```
In short, the whole argument `(H a a)` in Ltac seems to have disappeared.

Is this a known limitation, or is there some workaround for properly serializing `specialize`?

**Comment by ejgallego** [comment]
This is a bug, likely some debug code still around, should be easy to solve will look into it asap.

Essential question, what version are you using?

**Comment by ejgallego** [comment]
I am indeed waiting for the 8.9 beta to appear to release a new version that should significantly improve these kinds of problems. But yeah, the beta has been delayed a bit.

**Comment by palmskog** [comment]
I tried this on the head of the `v8.8` branch of serapi (last commit is `db59830dc`), using Coq 8.8.2. Also manifests in released serapi versions, e.g., 0.5.2.

**Comment by ejgallego** [comment]
Thanks @palmskog I will have a look ASAP.

By the way, did you see my post here https://gitter.im/coq-serapi/Lobby?at=5bc6001e82893a2f3bef0e05 ?

If you want to provide any info by mail it would be appreciated!

**Comment by ejgallego** [comment]
Fixed in #67 , there could be more instances as this is indeed an area it has changed in 8.9 so i didn't test too much.

**Comment by ejgallego** [comment]
Will do a release later today.
